nucliadb/src/nucliadb/ingest/orm/resource.py

```diff
@@ -1254,3 +1256,25 @@ def extract_field_metadata_languages(
     for _, splitted_metadata in field_metadata.metadata.split_metadata.items():
         languages.add(splitted_metadata.language)
     return list(languages)
+
+
+def _update_entities_dict(
+    target_entites_dict: Union[dict[str, str], ScalarMap[str, str]], field_metadata: FieldMetadata
+):
+    """
+    Update the entities dict with the entities from the field metadata.
+    Method created to ease the transition from legacy ner field to new entities field.
+    """
+    # Data Augmentation + Processor entities
+    # This will overwrite entities detected from more than one data augmentation task
+    # TODO: Change TrainMetadata proto to accept multiple entities with the same text
+    ents = {
```

Comment:
nit: prefer the full name (like `entities`) rather than a partial word like `ents`

Reply:
Fixed!